OptimizelySDKCore/OptimizelySDKCore/Optimizely.h

```diff
@@ -280,19 +257,16 @@ extern NSString * _Nonnull const OptimizelyNotificationsUserDictionaryExperiment
  */
 + (nullable instancetype)init:(nonnull OPTLYBuilderBlock)builderBlock;
 
-// TODO: remove when eventValue is deprecated
 /**
  * Track an event
  * @param eventKey The event name
  * @param userId The user ID associated with the event to track
  * @param attributes A map of attribute names to current user attribute values.
  * @param eventTags A map of event tag names to event tag values (string, number, or boolean)
- * @param eventValue The event value (e.g., revenue amount)
  */
 - (void)track:(nonnull NSString *)eventKey
```

Comment:
this API should be removed.

Reply:
I have removed eventValue param from this API.
Should I remove this whole API ? since it is of Optimizely class which defines the Optimizely SDK interface.